config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -170,6 +170,48 @@ presubmits:
           requests:
             cpu: 4
             memory: 6Gi
+  - name: pull-kubernetes-node-e2e-containerd-features
+    branches:
+    - master
+    always_run: false
+    optional: true
+    max_concurrency: 12
+    skip_report: false
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    annotations:
+      testgrid-create-test-group: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210622-762366a-master
+        args:
+        - --root=/go/src
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --repo=github.com/containerd/containerd=main
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --timeout=90
+        - --scenario=kubernetes_e2e
```

Comment:
Missing service account token I think: `        - "--service-account=/etc/service-account/service-account.json"`

Reply:
Ah - wasn't sure this was required when `preset-service-account` is set, because I didn't see it referenced otherwise in the scheduled jobs